webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    entry: ["./public/scripts/history.js", "./public/scripts/confetti.js", "./public/scripts/main.js","./public/scripts/keymaster.js" ],
+    target: 'node',
+    output: {
+        path: __dirname + "/dist",
+        filename: "bundle.js",
```

Comment:
Can you add please dist to .gitignore? That way we won't commit any of the generated files.